lib/repositories/stubs/stub_account_repository.dart

```diff
@@ -1,141 +1,38 @@
-    linked: false,
-  ));
-  listOfAccounts.add(Account(
-    phoneNumber: 'IN1233323987',
-    name: 'John Doe4',
-    balance: '100000',
-    accountNumber: '120980988878824',
-    bankName: 'Bank of India',
-    linked: false,
-  ));
-  listOfAccounts.add(Account(
-    phoneNumber: 'IN1233323987',
-    name: 'John Doe5',
-    balance: '100000',
-    accountNumber: '120980988878825',
-    bankName: 'Bank of India',
-    linked: false,
-  ));
-
-  return listOfAccounts;
-}
-
-List<Account> getOtherDummyAccounts() {
-  final List<Account> listOfAccounts = <Account>[];
-
-  listOfAccounts.add(Account(
-    phoneNumber: 'IN9999999999',
-    name: 'Mark Doe',
-    balance: '100000',
-    accountNumber: '120980988878818',
-    bankName: 'Bank of India',
-  ));
-  listOfAccounts.add(Account(
-    phoneNumber: 'IN9999999999',
-    name: 'Mark Doe1',
-    balance: '100000',
-    accountNumber: '12098098887822',
-    bankName: 'Bank of India',
-  ));
-  listOfAccounts.add(Account(
-    phoneNumber: 'IN1233323982',
-    name: 'Mark Doe2',
-    balance: '100000',
-    accountNumber: '120980988878838',
-    bankName: 'Bank of India',
-  ));
-  listOfAccounts.add(Account(
-    phoneNumber: 'IN1233323982',
-    name: 'Mark Doe3',
-    balance: '100000',
-    accountNumber: '120980988878848',
-    bankName: 'Bank of India',
-  ));
-  listOfAccounts.add(Account(
-    phoneNumber: 'IN1233323983',
-    name: 'Mark Doe4',
-    balance: '100000',
-    accountNumber: '120980988878858',
-    bankName: 'Bank of India',
-  ));
-  listOfAccounts.add(Account(
-    phoneNumber: 'IN1233323983',
-    name: 'Mark Doe5',
-    balance: '100000',
-    accountNumber: '120980988878868',
-    bankName: 'Bank of India',
-  ));
-
-  return listOfAccounts;
-}
-
-List<Account> getOtherAccountsByPhone(String phone) {
-  final List<Account> listOfAccounts = getOtherDummyAccounts();
-
-  // filter based on phone number
-  return listOfAccounts
-      .where(
-        (Account account) => account.phoneNumber == phone,
-      )
-      .toList();
-}
+    partyInfo: PartyIdInfo(
+      fspId: 'AJCICFQ1919',
+      partyIdentifier: 'IN1233323987',
+    ),
+    fspInfo: FspInfo(fspId: 'AJCICFQ1919', fspName: 'Axis Bank'),
+    consentId: '985',
+    userId: 'vXiSsQglsFYXqVkOHNKKFhnuAAI2',
+    sourceAccountId: 'bob.fspB',
+  ),
+];
 
 class StubAccountRepository implements IAccountRepository {
   @override
-  List<Account> getUserAccounts() {
-    return getMyDummyAccounts()
-        .where(
-          (Account element) => element.linked,
-        )
-        .toList();
-  }
-
-  @override
-  List<Account> getPayeeAccountsByPhone(String phoneNumber) {
-    return getOtherAccountsByPhone(phoneNumber);
-  }
-
-  List<Account> getOtherAccounts() {
-    return getOtherDummyAccounts();
+  Future<List<Account>> getUserAccounts(String userId) {
+    // final logger = getLogger('here');
+    // logger.e('stub');
+    return Future.value(accounts.where((element) => element.userId == userId).toList());
```

Comment:
Please don't filter anything here, it won't work for people who don't have access to your google account :)

Reply:
haha agreed